addons/web/static/src/views/kanban/kanban_dashboard.scss

```diff
@@ -185,14 +175,6 @@
         }
 
     }
-
-    .o_favorite {
```

Comment:
It's unlikely that this was affecting `project`'s  kanbans only.
Did you check all the modules using the favourite widget?

Reply:
@stefanorigano Yep, I checked all the kanbans with o_favorite, there aren't that many.